lib/node_runner.go

```diff
@@ -393,79 +403,29 @@ func (nr *NodeRunner) InitRound() {
 		"validators", nr.Policy().Validators(),
 	)
 
-	go nr.startRound()
+	nr.StartStateManager()
 }
 
-func (nr *NodeRunner) startRound() {
+func (nr *NodeRunner) StartStateManager() {
 	// check whether current running rounds exist
 	if len(nr.consensus.RunningRounds) > 0 {
 		return
 	}
 
-	nr.StartNewRound(0)
+	go nr.nodeRunnerStateManager.Start()
+	nr.nodeRunnerStateManager.ResetRound()
+	return
 }
 
 func (nr *NodeRunner) CalculateProposer(blockHeight uint64, roundNumber uint64) string {
 	return nr.proposerCalculator.Calculate(nr, blockHeight, roundNumber)
 }
 
-func (nr *NodeRunner) StartNewRound(roundNumber uint64) {
```

Comment:
This method modified to https://github.com/bosnet/sebak/pull/305/files#diff-92980aaa6564b8c3373783d7b7728063R83